admin/tools/docker/integration-tests/localhost/env.example.sh

```diff
@@ -0,0 +1,21 @@
+# VERSION can be ticket branch but with _ instead of /
```

Comment:
Yes file should only be sourced, so shebang isn't required. Comment added about this